rcsbapi/const.py

```diff
@@ -37,11 +32,40 @@ class Const:
     RCSB_SEARCH_API_QUERY_URL: str = "https://search.rcsb.org/rcsbsearch/v2/query"
     UPLOAD_URL: str = "https://user-upload.rcsb.org/v1/putMultipart"
     RETURN_UP_URL: str = "https://user-upload.rcsb.org/v1/download/"
-    #
+
+    SEARCH_API_SCHEMA_DIR = "search/resources"
+    SEARCH_API_SCHEMA_BASE_URL = "http://search.rcsb.org/rcsbsearch/v2/metadata/"
+    STRUCTURE_SCHEMA_FILE_NAME = "structure_schema.json"
+    CHEMICAL_SCHEMA_FILE_NAME = "chemical_schema.json"
+    SEARCH_API_SCHEMA_FILE_TO_ENDPOINT = MappingProxyType({
+        STRUCTURE_SCHEMA_FILE_NAME: "schema",
+        CHEMICAL_SCHEMA_FILE_NAME: "chemical/schema",
+    })
+    STRUCTURE_ATTRIBUTE_SCHEMA_URL: str = SEARCH_API_SCHEMA_BASE_URL + SEARCH_API_SCHEMA_FILE_TO_ENDPOINT[STRUCTURE_SCHEMA_FILE_NAME]
+    STRUCTURE_ATTRIBUTE_SCHEMA_FILE: str = SEARCH_API_SCHEMA_DIR + "/" + STRUCTURE_SCHEMA_FILE_NAME
+    CHEMICAL_ATTRIBUTE_SCHEMA_URL: str = SEARCH_API_SCHEMA_BASE_URL + SEARCH_API_SCHEMA_FILE_TO_ENDPOINT[CHEMICAL_SCHEMA_FILE_NAME]
+    CHEMICAL_ATTRIBUTE_SCHEMA_FILE: str = SEARCH_API_SCHEMA_DIR + "/" + CHEMICAL_SCHEMA_FILE_NAME
+
     # Data API constants
     DATA_API_ENDPOINT: str = "https://data.rcsb.org/graphql"
-    #
-    SINGULAR_TO_PLURAL: Dict[str, str] = MappingProxyType({
+    DATA_API_SCHEMA_DIR: str = "data/resources"
+    DATA_API_SCHEMA_BASE_URL: str = "https://data.rcsb.org/rest/v1/schema/"
+    DATA_API_SCHEMA_FILE_TO_ENDPOINT = MappingProxyType({
+        "entry.json": "entry",
+        "polymer_entity.json": "polymer_entity",
+        "branched_entity.json": "branched_entity",
+        "nonpolymer_entity.json": "nonpolymer_entity",
+        "polymer_entity_instance.json": "polymer_entity_instance",
+        "branched_entity_instance.json": "branched_entity_instance",
+        "nonpolymer_entity_instance.json": "nonpolymer_entity_instance",
+        "assembly.json": "assembly",
+        "chem_comp.json": "chem_comp",
+        "pubmed.json": "pubmed",
+        "uniprot.json": "uniprot",
+        "drugbank.json": "drugbank",
+    })
```

Comment:
Can you make use of these (where applicable) in `test_data_schema.py`? (E.g., using the similar join method described earlier)